aten/src/ATen/native/cuda/IndexKernel.cu

```diff
@@ -324,14 +324,24 @@ void masked_scatter_cuda_impl(Tensor& self, const Tensor& mask, const Tensor& so
   // number of elements available in `src`
   TORCH_CHECK(totalElements <= srcSize, "source nElements must be == mask `1` elements");
 
+
+#if CUDA_VERSION >= 1120
```

Comment:
For rocm compatibility, we may change this to 

```cpp
#if defined(CUDA_VERSION) && CUDA_VERSION >= 11020
...
```

Reply:
Sure! Thanks!